lib/screens/v2/candidate_generator/widgets/train_crowding.ex

```diff
@@ -88,4 +107,17 @@ defmodule Screens.V2.CandidateGenerator.Widgets.TrainCrowding do
     localized_alert.alert.effect in [:suspension, :shuttle] and
       LocalizedAlert.location(localized_alert) in [:boundary_downstream, :boundary_upstream]
   end
+
+  defp log_crowding_info(prediction, crowding_config, true, screen_id, triptych_pane) do
+    crowding_levels =
+      prediction.vehicle.carriages
+      |> Enum.sort_by(& &1.car_number)
+      |> Enum.map_join(",", & &1.occupancy_status)
+
+    Logger.info(
+      "[train_crowding car_crowding_info] screen_id=#{screen_id} triptych_pane=#{triptych_pane} station_id=#{crowding_config.station_id} direction_id=#{crowding_config.direction_id} trip_id=#{prediction.trip.id} prediction_id=#{prediction.id} vehicle_id=#{prediction.vehicle.id} car_crowding_levels=#{crowding_levels}"
```

Comment:
Which are the primary values that we need in splunk? `vehicle_id` is here, but not in the other log statement, so I'm wondering which values will help us most in the logs: prediction id, vehicle id, trip id?

Reply:
I think the amount I included is overkill. We really only need `screen_id`, `triptych_pane`, `trip_id`, and `car_crowding_levels`. The rest I included can either be easily found using the `screen_id` or will not be needed (like `prediction_id`).